molo/profiles/forms.py

```diff
@@ -201,10 +307,31 @@ def __init__(self, *args, **kwargs):
         self.fields['email'].required = (
             profile_settings.email_required and
             profile_settings.show_email_field)
+        self.fields['alias'].required = (
+            profile_settings.activate_display_name and
+            profile_settings.capture_display_name_on_reg and
```

Comment:
@Ntu2koTrevor we don't need to check for the `capture_...` it can be required even if is not in registration